src/main/java/org/breedinginsight/brapps/importer/model/imports/experimentObservation/ExperimentImportService.java

```diff
@@ -18,36 +18,28 @@
 package org.breedinginsight.brapps.importer.model.imports.experimentObservation;
 
 import lombok.extern.slf4j.Slf4j;
-import org.breedinginsight.brapps.importer.model.ImportUpload;
-import org.breedinginsight.brapps.importer.model.imports.BrAPIImport;
-import org.breedinginsight.brapps.importer.model.imports.BrAPIImportService;
-import org.breedinginsight.brapps.importer.model.response.ImportPreviewResponse;
+import org.breedinginsight.brapps.importer.model.imports.DomainImportService;
 import org.breedinginsight.brapps.importer.services.processors.ExperimentProcessor;
-import org.breedinginsight.brapps.importer.services.processors.Processor;
 import org.breedinginsight.brapps.importer.services.processors.ProcessorManager;
-import org.breedinginsight.model.Program;
-import org.breedinginsight.model.User;
-import tech.tablesaw.api.Table;
+import org.breedinginsight.brapps.importer.services.processors.experiment.ExperimentWorkflowNavigator;
 
 import javax.inject.Inject;
 import javax.inject.Provider;
 import javax.inject.Singleton;
-import java.util.List;
 
 @Singleton
 @Slf4j
-public class ExperimentImportService implements BrAPIImportService {
+public class ExperimentImportService extends DomainImportService {
 
     private final String IMPORT_TYPE_ID = "ExperimentImport";
 
-    private final Provider<ExperimentProcessor> experimentProcessorProvider;
-    private final Provider<ProcessorManager> processorManagerProvider;
-
+    // TODO: delete processor fields once WorkflowNavigator is used
     @Inject
-    public ExperimentImportService(Provider<ExperimentProcessor> experimentProcessorProvider, Provider<ProcessorManager> processorManagerProvider)
+    public ExperimentImportService(Provider<ExperimentProcessor> experimentProcessorProvider,
+                                   Provider<ProcessorManager> processorManagerProvider,
+                                   ExperimentWorkflowNavigator workflowNavigator)
     {
-        this.experimentProcessorProvider = experimentProcessorProvider;
-        this.processorManagerProvider = processorManagerProvider;
+        super(experimentProcessorProvider, processorManagerProvider, workflowNavigator);
     }
```

Comment:
I merged into [BI-2134](https://github.com/Breeding-Insight/bi-api/pull/359) the changes from BI-2132 for `ExperimentFileImportTest` and was able to implement this TODO. So, there isn't any additional work required, you can just accept the changes during the merge.